nussl/datasets/hooks.py

```diff
@@ -187,22 +190,22 @@ def get_items(self, folder):
         ])
         return items
 
-    def get_mix_and_sources(self, item):
+    def get_mix_and_sources(self, item, **kwargs):
```

Comment:
Why do these all need to be kwargs now?

Reply:
passing kwargs allows these functions to interact with the AudioSignal object by passing along the desired parameters. In this specific case it lets the SalientExcerptMixSourceFolder._get_mix to read an audio file with a specified offset and duration.